Dependencies/AndroidExtensions/Source/JavaWrappers.cpp

```diff
@@ -164,6 +164,36 @@ namespace java::net
     }
 }
 
+namespace android
+{
+    jstring ManifestPermission::getPermissionName(const char* permissionName)
```

Comment:
Seems like this should be private, and we should just expose functions like `CAMERA()` that call this function with the right hard coded constant strings (names of Java class fields). This would be a better 1:1 mapping of the underlying Java classes.

Reply:
done